client/client_test.go

```diff
@@ -3347,12 +3371,16 @@ func testNoTarOCIIndexMediaType(t *testing.T, sb integration.Sandbox) {
 
 func testOCIIndexMediatype(t *testing.T, sb integration.Sandbox) {
 	workers.CheckFeatureCompat(t, sb, workers.FeatureOCIExporter)
-	requiresLinux(t)
 	c, err := New(sb.Context(), sb.Address())
 	require.NoError(t, err)
 	defer c.Close()
 
-	st := llb.Image("busybox:latest").Run(llb.Shlex(`sh -c "echo -n hello > hello"`))
+	imgName := integration.UnixOrWindows("busybox:latest", "nanoserver:latest")
+	cmdStr := integration.UnixOrWindows(
+		`sh -c "echo -n hello > hello"`,
```

Comment:
nit: doesn't look like the file contents are relevant to the the the test so this can simply be a shell prefix change and Windows will write `-n hello` to the file since `-n` switch isn't supported or we can drop the `-n` altogether

Reply:
though that could be misleading, just wanted both files on Windows and Linux to be the same -- empty files. Also someone reading this later without context, might think the intention was to provide the unsupported `-n` switch to `echo` on Windows.